README.md

```diff
@@ -19,6 +19,18 @@ the UI framework used in [Fuse](https://www.fusetools.com/) apps.
 * `build.sh` runs stuff (which downloads uno if needed), and builds all packages.
 * `test.sh` runs all tests.
 
+
+### Fuse
+
+If you want to use a self-built fuselibs installation with an installed Fuse, you can do this through a `.unoconfig`-file, something like this:
```

Comment:
@sebbert: Yeah, I wrote some rant about that in the cover-letter of this PR (above). I'm still a little bit torn if we should advice people about the home-directory, as that affects all projects. We could also advice them to put the `.unoconfig` in their project folder. And finally, we could just reinstate the top-level .unoconfig in the fuselibs repo, and tell people to run uno from inside there.

I'm not sure which option is better.